.travis.yml

```diff
@@ -95,7 +95,9 @@ install:
       travis_retry travis_wait \
           mvn install -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode --show-version;
     elif [ "${TRAVIS_BRANCH}" == "${COVERITY_BRANCH_NAME}" ] || ! [ -z "${COVERITY_ONLY:-}" ]; then
-      echo "Skipping module build on Coverity branch/job";
+      echo "Building all modules for test-compile coverage, but skipping Coverity upload";
+      travis_retry travis_wait \
+          mvn install -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode --show-version;
```

Comment:
I verified that `mvn install -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode --show-version`  will catch the compilation errors.

The problem with running the full dist is if I put it in the normal block -Pcoverage gets added and then fails because it can't find jacoco-combined.exec. 

I could've tweaked the pom on janusgraph-all or added another job to the matrix and another conditional that didn't add -Pcoverage for that single job. Since this also catches the errors mentioned in issue #1122 it seemed like the path of least resistance.

let me know if you still want me to have it run a full dist build instead.

Reply:
Makes sense, thanks. As a final check after this is merged we can update the `coverity` branch to ensure the scan/upload still works as expected.